### ISSUE TITLE
Markdown formatting support

### ISSUE THREAD

**Issue by harefoot55** [madskristensen/MarkdownEditor2022#32]
Would it be possible to add document formatting support (text wrapping) for Markdown documents? This feature was mentioned on your product road map a few years ago and I wanted to add an upvote for it. Thanks for continuing to develop these projects for the community.
![screencap_20220310_112028](https://user-images.githubusercontent.com/16060436/158901632-5bed77f3-95b1-48dd-a43c-1ab56aca4f4f.png)

**Comment by madskristensen** [comment]
Can you use Visual Studio's word wrapping feature? That's what I use.

![image](https://user-images.githubusercontent.com/1258877/160009653-c5f2f488-ee47-41c8-b2ee-8cd020ca3436.png)

**Comment by madskristensen** [comment]
ping @harefoot55

**Comment by harefoot55** [comment]
Thanks....I tried that, but I don't see a way to set it to wrap at 80 characters. Is there a way to customize where/when it wraps?

**Comment by madskristensen** [comment]
No, it wraps when there's no more horizontal space and not after a fixed number of characters. To vote for that feature, go here: https://developercommunity.visualstudio.com/t/Support-max_line_length-in-editorconfig/567214?space=8&q=editorconfig+max

Since this is tracked as a feature request on the Developer Community website, can we close it here?

**Comment by harefoot55** [comment]
Thanks for your comments.
